data/net/minecraft/commands/arguments/item/FunctionArgument.mapping

```diff
@@ -15,6 +15,7 @@ CLASS net/minecraft/commands/arguments/item/FunctionArgument
 		ARG 0 tag
 	METHOD lambda$static$1 (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
 		ARG 0 function
+	METHOD parse (Lcom/mojang/brigadier/StringReader;)Ljava/lang/Object;
```

Comment:
Please revert this change and the others similar to it in the same package, as this adds nothing. (This will likely be removed by a `sanitizeData` pass.)